cmd/rancher-desktop-guestagent/main.go

```diff
@@ -140,7 +141,7 @@ func main() {
 		forwarder := forwarder.NewVTunnelForwarder(*vtunnelAddr)
 		portTracker = tracker.NewVTunnelTracker(forwarder, wslAddr)
 	} else {
-		portTracker = tracker.NewAPITracker(tracker.GatewayBaseURL)
+		portTracker = tracker.NewAPITracker(tracker.GatewayBaseURL, *adminInstall)
```

Comment:
So `adminInstall` is mutually exclusive with `enablePrivilegedService`, then?
(No need to add a check for it at this stage, I'm just clarifying.)